src/elements/asset.js

```diff
@@ -0,0 +1,26 @@
+export default function(namespace) {
```

Comment:
Just a thought, but wondering if you could make each of these files their own plugins, and then have one main function that calls `use` for each of them. Just a thought.